challenge_gpsr/src/gpsr.py

```diff
@@ -191,7 +193,7 @@ def wait_to_be_called(self):
                     rospy.logwarn("[GPSR] Timeout_count: {}".format(self.timeout_count))
 
     def heard_correct(self, sentence):
-        self.robot.speech.speak('I heard %s, is this correct?' % sentence)
+        self.robot.speech.speak('I heard %s, is this correct? Please speak loudly into the microphone and move away afterwards!' % sentence)
```

Comment:
Same

Reply:
Shortened by introducing the tc_fuckup time once again